docs/source/normalization.rst

```diff
@@ -0,0 +1,51 @@
+#############
```

Comment:
This doc sounds more like a design doc than a OS usage documentation. Maybe save this for a `release-3-3-0` release notes?

Reply:
LOL, yeah. It is on my todo list to add configuration and deployment instructions in following two PRs. Then this page won't look just like a design doc.

Reply:
agree w/ derek here - remove these and set aside for later. I'd focus on docs last for this feature because of the likelihood to change before being finalized